tests/functional/spec/features/calculated_summary/new_calculated_summary_repeating_blocks.spec.js

```diff
@@ -0,0 +1,160 @@
+    await repeatingAnswerChangeLink(1).click();
+    await $(TransportRepeatingBlock1Page.previous()).click();
+    await expect(await browser.getUrl()).to.contain(CalculatedSummarySpendingPage.pageName);
+  });
+
+  it("Given I edit one of my answers, I am taken back to the calculated summary and the new total is correct", async () => {
+    await repeatingAnswerChangeLink(1).click();
+    await $(TransportRepeatingBlock1Page.transportCost()).setValue(60);
+    await $(TransportRepeatingBlock1Page.submit()).click();
+    await expect(await browser.getUrl()).to.contain(CalculatedSummarySpendingPage.pageName);
+    await expect(await $(CalculatedSummarySpendingPage.calculatedSummaryTitle()).getText()).to.contain(
+      "We calculate the total monthly expenditure on transport to be £580.00. Is this correct?"
+    );
+    await assertSummaryValues(["£100.00", "£60.00", "£5.00", "£0.00", "£265.00", "£100.00", "£50.00"]);
+    await $(CalculatedSummarySpendingPage.submit()).click();
+  });
+
+  it("Given I edit an answer for the second calculated summary, I am taken back to the second calculated summary and the new total is correct", async () => {
+    await repeatingAnswerChangeLink(2).click();
+    await $(TransportRepeatingBlock2Page.transportCount()).setValue(12);
+    await $(TransportRepeatingBlock2Page.submit()).click();
+    await expect(await browser.getUrl()).to.contain(CalculatedSummaryCountPage.pageName);
+    await expect(await $(CalculatedSummaryCountPage.calculatedSummaryTitle()).getText()).to.contain(
+      "We calculate the total journeys made per month to be 24. Is this correct?"
+    );
+    await assertSummaryValues(["10", "2", "12"]);
+    await $(CalculatedSummaryCountPage.submit()).click();
+  });
+
+  it("Given I remove one of the list items, I am taken back to each calculated summary in turn to confirm the new totals", async () => {
+    await $(SectionOnePage.transportListRemoveLink(1)).click();
+    await $(RemoveTransportPage.yes()).click();
+    await $(RemoveTransportPage.submit()).click();
+    await expect(await browser.getUrl()).to.contain(CalculatedSummarySpendingPage.pageName);
+    await expect(await $(CalculatedSummarySpendingPage.calculatedSummaryTitle()).getText()).to.contain(
+      "We calculate the total monthly expenditure on transport to be £515.00. Is this correct?"
+    );
+    await assertSummaryValues(["£100.00", "£0.00", "£265.00", "£100.00", "£50.00"]);
+    await $(CalculatedSummarySpendingPage.submit()).click();
+    await expect(await browser.getUrl()).to.contain(CalculatedSummaryCountPage.pageName);
+    await expect(await $(CalculatedSummaryCountPage.calculatedSummaryTitle()).getText()).to.contain(
+      "We calculate the total journeys made per month to be 14. Is this correct?"
+    );
+    await assertSummaryValues(["2", "12"]);
+  });
+
+  it("Given I enter section 2, the calculated summary from section 1 is correctly used to validate my answer", async () => {
```

Comment:
We should do one of these once a list item is added/removed. We need to also assert the status of the hub to verify the dependencies are working properly.

Reply:
Run into a problem with this one, currently calculated summary progress doesn't update by removing an answer from the path, only changing one, looking into this

Reply:
As per chat, list collector on/off path doesn't directly update CS progress but add/removing items does as I've changed the extra dependency on the remove block to be for add as well

Added a test for Hub progress when adding a new item and a test for correct values on the CS when list collector is removed from the path